resources/examples/manager/SampleAssetManager/python/SampleAssetManager/SampleAssetManagerInterface.py

```diff
@@ -43,5 +43,13 @@ class SampleAssetManagerInterface(ManagerInterface):
     def identifier():
         return "org.openassetio.examples.manager.sam"
 
+    def initialize(self, hostSession):
+        pass
```

Comment:
[fix] New to me, but `pylint` will moan at you with `unnecesary-pass`

Reply:
Interesting, my pylint doesn't warn about that... it's certainly not unnecessary

Reply:
Got to the bottom of it [here](https://github.com/PyCQA/pylint/issues/2208). I did think it was odd that `pass` wasn't needed. I didn't know that the docstrings counted!